docs/source/specification.md

```diff
@@ -63,18 +63,102 @@ Together, these provide a framework that {ref}`TRE operator <infrastructure_role
+
+#### Maintaining public trust
+
+##### Statement
+
+TREs holding public data should build and maintain the trust of data subjects and any other impacted individuals, groups, communities and organisations by protecting privacy, keeping data secure and being transparent about their work.
+
+##### Rationale
+
+Maintaining trust in TREs to hold and use data is essential to prevent backlash or resistance to the use of data.
+This can include maintaining the trust of members of the public who are impacted by research, as well as the trust of commercial data providers.
+
+In the case of public sector data, public engagement work has indicated there is widespread support for the use of regulated and ethical TREs working for the public benefit.
+Consulting impacted parties, including the public, can help ensure TREs are being used for positive, impactful and agreed-upon purposes.
+Being transparent about the data held and the projects or organisations who access the data can also help maintain trust.
+
+##### Implications
+
+- Being as transparent as possible is key to building trust. TREs holding public data should practice transparency, for instance making adherence to any accreditation body, or framework (for instance the Five Safes) publicly available information, and making information on the projects or organisations which access their data available to impacted parties.
+- A strong public engagement programme takes time and resource. TREs holding public data should consider allocating specific staffing to public engagement activities.
+- TREs open to consultation by impacted parties should be auditable, and impacted parties should be part of any decision making processes. This may include the provision of documentation and educational resources for a diverse audience.
+- Access to public sector data should be reviewed by an independent panel and follow agreed-upon governance to ensure projects using this data are in the public benefit, and provide clarity around any commercial access.
+
+#### Observability
+
+##### Statement
+
+Human initiated and automated processes resulting in change within the TRE should be observable.
+
+##### Rationale
+
+System/process observability is key to understanding whether your policies and controls are actually doing what is intended.
+
+It also allows operators to continuously improve their systems and processes, measure their effectiveness, and identify the causes of incidents. Data can also be made available to other parties such as auditors, data subjects and data providers as part of the assurance process.
+
+This applies to both technical systems and policies/processes.
+
+##### Implications
+
+- In order to understand what is happening within the TRE, both automated and human initiated processes should generate sufficient data. Any generated data should follow standards for provenance and transparency for audit trails.
```

Comment:
"generate sufficient data" sounds quite vague. Can we be more specific- e.g. Audit logs?

Reply:
```suggestion
- In order to understand what is happening within the TRE, both automated and human initiated processes should generate sufficient data, for instance through audit logs. Any generated data should follow standards for provenance and transparency for audit trails.
```